src/inpod.rs

```diff
@@ -66,14 +66,11 @@ impl WorkloadUid {
     }
 }
 
-#[derive(Debug, Clone)]
-struct WorkloadInfo {
-    workload_uid: WorkloadUid,
-}
 #[derive(Debug)]
 pub struct WorkloadData {
     netns: std::os::fd::OwnedFd,
-    info: WorkloadInfo,
+    workload_uid: WorkloadUid,
+    workload_info: Option<istio::zds::WorkloadInfo>,
```

Comment:
is it `Option` forever or just because Istio doesn't yet send it?

Reply:
probably forever - it will also be an option for VMs

Reply:
though i guess for VMs this can come from config.. so we'll see when we get there.

Reply:
VMs don't use inpod though, right?

I think it needs to be an `Option` in the rbac enforcement but maybe not here.

Anyways, not a big deal, was mostly checking my understanding

Reply:
VMs won't use inpod, but conceptionally may want something similar. 

yea right now it needs to be an option otherwise it will break istio integration tests.
Will soon follow up with istio/istio PR to fill in the info and we can decide if to remove the option from here once its all settled.